docs/connected_orgs.rst

```diff
@@ -1,14 +1,133 @@
 
-Connect a sandbox org
----------------------
+.. note:: A different setup is required to connect to orgs in the context of an automated build. See `continuous integration<TODO>`_ for more information.
 
-Global orgs
------------
 
-Use a custom connected app
+
+The ``org connect`` Command
+---------------------------
+
+To connect to a persistent org:
+
+.. code-block:: console
+
+    $ cci org connect <org_name>
+
+This command automatically opens a browser window pointed to a Salesforce login page. The provided ``<org_name>`` is the org that CumulusCI directs you to log into.
+
+.. note::
+    Connecting an org via ``cci org connect`` does *not* expose that org to the Salesforce CLI.
+
+If your org has a custom domain, use the ``--login-url`` option along with the org's login url.
+
+.. code-block:: console
+
+    cci org connect <org_name> --login-url https://example.my.domain.salesforce.com
+
+
+Production and Developer Edition Orgs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No options are needed for these org types. Just run the same command you normally would to connect to a persistent org.
+
+.. code-block:: console
+
+    $ cci org connect <org_name>
+
+
+Sandboxes
+^^^^^^^^^
+For sandboxes, pass the ``--sandbox`` flag along with the org name.
+
+.. code-block:: console
+
+    $ cci org connect <org_name> --sandbox
+
+.. note:: The ``--sandbox`` flag can also be used for connecting a scratch org created externally to CumulusCI.
+
+
+
+Verify Your Connected Orgs
 --------------------------
+
+Run ``cci org list`` to see your org listed under the "Connected Org" table.
+
+    Example: A single persistent org connected to CumulusCI with the name ``devhub``.
+
+.. code-block:: yaml
+
+    $ cci org list
+
+    ┌Scratch Orgs──────────────┬─────────┬──────┬─────────┬──────────────┬────────┐
+    │ Name                     │ Default │ Days │ Expired │ Config       │ Domain │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ dev                      │         │ 7    │ X       | dev          │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ feature                  │         │ 1    │ X       | feature      │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ prerelease               │         │ 1    │ X       | prerelease   │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ qa                       │         │ 7    │ X       | qa           │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ release                  │         │ 1    │ X       | release      │        │
+    └──────────────────────────┴─────────┴──────┴─────────┴──────────────┴────────┘
```

Comment:
I cleaned up the formatting of this table, but I had to get read of the original X's that had a bit more flourish. We can put them back in, but they will mess up the alignment of the table borders and columns.